### PR TITLE
stupid way to display logs and stuff for debugging

### PR DESCRIPTION
**Pull request by indig**
This is pretty dumb but pretty helpful to see logs without SSHing. It's probably broken and bad for security etc. but it's nice to be able to quickly look at logs 🤷‍♂ There's probably a better way to do this.

### CODE REVIEW

File: polynote-server/src/main/scala/polynote/server/Server.scala [hunk 1]
```diff
@@ -48,6 +63,8 @@ trait Server extends IOApp with Http4sDsl[IO] with KernelLaunching {
       case req @ GET -> "notebook" /: path :? DownloadMatcher(Some("true")) =>
         IO(logger.info(s"Download request for ${req.pathInfo} from ${req.remoteAddr}")) *> downloadFile(path.toList.mkString("/"), req, config)
       case req @ GET -> "notebook" /: _ => serveFile(indexFile, req, watchUI)
+      case req @ GET -> Root / "logs" => showDir(config.storage.logs, req)
+      case req @ GET -> Root / "logs" / path => showFile(s"${config.storage.logs}/$path", req)
```
Comment (jeremyrsmith) [inline polynote-server/src/main/scala/polynote/server/Server.scala]: As long as you tested this against stuff like `/logs/../../../../etc/passwd` 😛

Comment (jeremyrsmith) [reply polynote-server/src/main/scala/polynote/server/Server.scala]: Err, I guess the notebook download thingy would have the same issue in that case. We should try to use both of those as attack vectors and see if it's reasonably OK.

Comment (indig) [reply polynote-server/src/main/scala/polynote/server/Server.scala]: Yeah I can try it out. maybe only allow relative paths or something.

**Comment by indig** [comment]
Nope.

**Pull request closed without merging**